src/applications/disability-benefits/all-claims/tests/pages/uploadPtsdDocuments781a.unit.spec.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { mount } from 'enzyme';
+
+import {
+  DefinitionTester, // selectCheckbox
+} from '../../../../../platform/testing/unit/schemaform-utils.jsx';
+import formConfig from '../../config/form.js';
+
+describe('781a record upload', () => {
+  const page = formConfig.chapters.disabilities.pages.uploadPtsdDocuments781a;
+  const { schema, uiSchema, arrayPath } = page;
+
+  it('should render', () => {
+    const form = mount(
+      <DefinitionTester
+        arrayPath={arrayPath}
+        pagePerItemIndex={0}
+        definitions={formConfig.defaultDefinitions}
+        schema={schema}
+        data={{
+          'view:selectablePtsdTypes': {
+            'view:assaultPtsdType': true,
+          },
+        }}
+        uiSchema={uiSchema}
+      />,
+    );
+
+    expect(form.find('input').length).to.equal(1);
+  });
+
+  it('should submit without an upload', () => {
```

Comment:
Same question I had for the 781; if they select the option to upload a document, does it make sense to not make this required?

Reply:
So actually just got confirmation on these, 781/781a they do want to make these required now. I'll get this updated.

Reply:
Good to hear about the required bit, thanks!

As far as the max upload size goes, that's...unfortunate. :grimacing: I saw you mentioned that yesterday in slack. I just brought it up with our team to make sure we don't lose sight of it. Thanks for testing it!